tests/reference/llvm-test_c_interop1-9657eef.stderr

```diff
@@ -0,0 +1,4 @@
+code generation error: asr_to_llvm: module failed verification. Error:
+Stored value type does not match pointer operand type!
+  store %array* %y, %array.0** %yptr1, align 8
+ %array.0*
```

Comment:
I would not be adding LLVM test failures here, because they might depend on LLVM version and ultimately our LLVM backend should never fail like this. It should fail by giving LFortran codegen error (which we can and should test) and ultimately this should be a semantic error, already in AST->ASR.

Reply:
Okay, I'll open an issue for the same.